### PR TITLE
escape html

### PR DESCRIPTION
**Pull request by ArtificialOwl**

### CODE REVIEW

**Review by sorbaugh — APPROVED** [review]

File: js/files/circles.files.list.js [hunk 1]
```diff
@@ -124,11 +124,20 @@
...
+						return this.escapeHTML(circle.name);
+					},
+
+					escapeHTML: function (text) {
+						return text.toString()
+							.split('&').join('&amp;')
+							.split('<').join('&lt;')
+							.split('>').join('&gt;')
+							.split('"').join('&quot;')
+							.split('\'').join('&#039;');
```
Comment (Fenn-CS) [inline js/files/circles.files.list.js:140]:
```suggestion
  						const parser = new DOMParser();
  						const doc = parser.parseFromString(text, 'text/html');
  						return doc.body.textContent;
```

Comment (Fenn-CS) [reply js/files/circles.files.list.js:140]: This would escape any chars not thought about and handle more edge cases, it however looks okay

**Review by Fenn-CS — APPROVED** [review]
Add a similar function is shorter and would handle all edge cases for your testing and consideration but his looks good too! Thanks

**Pull request merged**